src/components/Team/CardContainer.css

```diff
@@ -1,21 +1,19 @@
 .cards {
-    display: inline-flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    width: 100%;
- }
+	display: inline-flex;
```

Comment:
We should consider refactoring to use `styled-components`, eventually.

Reply:
agreed